hooks/01_rpi-firmware.yaml

```diff
@@ -0,0 +1,6 @@
+name: "Raspberry Pi post disk hook"
+stages:
+    after-disk:
+    - name: "Copy firmware to EFI partition"
+      commands:
+      - cp -r /iso/build/recovery.img.root/boot/vc/* /iso/build/efi/
```

Comment:
I am not entirely sure which are those paths... I do not remember all the details of the build env now. But probably we could add some sort of clause depending on the existence of contents in /boot/vc/*. I assume those are firmware specific files.

Reply:
See lines 104ff of [generate-rpi-image.sh](https://build.opensuse.org/package/view_file/isv:Rancher:Elemental:Dev/elemental-post-build-generate-rpi-image/generate-rpi-image.sh?expand=1)

* /boot/vc/* is the current (working, supported) firmware
* the [UEFI firmware](https://build.opensuse.org/package/show/isv:Rancher:Elemental:Dev/raspberrypi-firmware-uefi) is not fully working on Raspberry (yet ?!)

Reply:
The probably we could add the same validation here.

Something like 

```yaml
    after-disk:
    - if: '[ -d "/iso/build/recovery.img.root/boot/vc" ]'
      name: "Copy firmware to EFI partition"
      commands:
      - cp -r /iso/build/recovery.img.root/boot/vc/* /iso/build/efi/
```

Reply:
Yep good idea! We currently get an ugly error log on x86 in the seedimage build pod.. I will add the conditional!